project.clj

```diff
@@ -102,10 +102,7 @@
                                                                                                         vec))))))
                                                        'user/nedap-ensure-exclusions)]}
 
-             :cljs-old              {:dependencies [[cljfmt "0.6.5"]
-                                                    [com.stuartsierra/component "0.4.0"]
-                                                    [integrant "0.8.0"]
```

Comment:
all part of `provided`. 

This change means you can no longer use `-provided,+cljs-old`. But I suppose that makes sense. The `clj-old`-profile now only adds an old clj 😄